packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import '../../../shared/utils.dart';
+import 'logging_controller_v2.dart';
+import 'logging_table_row.dart';
+
+class LoggingTableModel extends ChangeNotifier {
+  LoggingTableModel() {
+    _worker = InterruptableChunkWorker(
+      callback: getRowHeight,
+      progressCallback: (progress) => _cacheLoadProgress.value = progress,
+    );
+  }
+
+  final List<LogDataV2> _logs = [];
+  final List<LogDataV2> _filteredLogs = [];
+  final Set<int> _selectedLogs = <int>{};
+
+  final Map<int, double> cachedHeights = {};
+  final Map<int, double> cachedOffets = {};
+
+  late final InterruptableChunkWorker _worker;
+
+  /// Represents the state of reloading the height caches.
+  ///
+  /// When null, then the cache is not loading.
+  /// When double, then the value is represents how much progress has been made.
+  ValueListenable<double?> get cacheLoadProgress => _cacheLoadProgress;
+  final _cacheLoadProgress = ValueNotifier<double?>(null);
```

Comment:
this notifier needs to be disposed. Add a `dispose` method, and then call it from the appropriate place in from the owner of the LoggingTableModel object